ui/selectors/selectors.js

```diff
@@ -1794,6 +1794,10 @@ export function getSortedAnnouncementsToShow(state) {
  * @returns {{networkId: string}[]}
  */
 export function getOrderedNetworksList(state) {
+  console.log(
+    'state.metamask.orderedNetworkList ----',
```

Comment:
A console.log that slipped

Reply:
fixed [735e2ef](https://github.com/MetaMask/metamask-extension/pull/28006/commits/735e2ef14d7817bf18e3af635709cc543f153919)